src/FreeformIdeas.js

```diff
@@ -4,18 +4,28 @@ import { Idea } from './Idea';
 import { ItemTypes } from './ItemTypes';
 import update from 'immutability-helper';
 import { v4 as uuidv4 } from 'uuid';
-import Popup from './Popup';
+import Popup from './AddPopup';
+import EditPopup from './EditPopup';
 
 
 const FreeForm = ({ hideSourceOnDrag }) => {
```

Comment:
Is `hideSourceOnDrag` ever going to change? Maybe we should just inline its value.

Reply:
I believe that hideSourceOnDrag is just the differentiation of whether or not the original placement is shown when dragging. Unless its decided that there will be some components that drag with source shown, then yes. My gut says there wont be want to show the source later on